devices/awox.js

```diff
@@ -8,6 +68,23 @@ module.exports = [
         description: 'LED white',
         extend: extend.light_onoff_brightness(),
     },
+    {
+        fingerprint: [
+            {type: 'Router', manufacturerName: 'AwoX', modelID: 'TLSR82xx', endpoints: [
+                {ID: 1, profileID: 260, deviceID: 2028, inputClusters: [0, 3, 4, 4096], outputClusters: [0, 3, 4, 5, 6, 8, 768, 4096]},
+                {ID: 3, profileID: 4751, deviceID: 2048, inputClusters: [65360, 65361], outputClusters: [65360, 65361]},
+            ]},
+        ],
+        model: '33952',
+        vendor: 'AwoX',
+        description: 'Remote controller',
+        fromZigbee: [fz.command_on, fzLocal.colors, fzLocal.refresh, fzLocal.refreshColored, fz.command_off,
+            fz.command_step, fz.command_move, fz.command_stop, fz.command_recall, fz.command_step_color_temperature],
+        toZigbee: [],
+        exposes: [e.action(['on', 'off', 'button_red', 'button_refresh', 'button_refresh_colored', 'button_blue', 'button_yellow',
+            'button_green', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down', 'brightness_stop',
```

Comment:
Last note (sorry I didn't mention this initially). It is better to remove the `button_` prefix, this make it more consistent with the other actions (e.g. `on` isn't called `button_on`)

Reply:
No problem, fixed :) 
Are these rules perhaps written down in some file?
Sorry for my ignorance, I didn't search before.